packages/office-ui-fabric-react/src/components/Rating/Rating.tsx

```diff
@@ -90,12 +90,20 @@ export class Rating extends BaseComponent<IRatingProps, IRatingState> {
     this.setState({
       focusedRating: value
     } as IRatingState);
+
+    if (this.props.onFocus) {
```

Comment:
Since setState is asynchronous it is possible that the state change won't complete before this element get's focus. If that's not a problem then no changes are necessary, otherwise you'll want to put the onfocus part in the setstate clalback like `this.setState(obj, ()=>callback)`